### DISCUSSION TITLE
配置mock的exclude无效？

### DISCUSSION THREAD

**Discussion by calmchang** [umijs/umi#7427] [Q&A]
在config/config.ts内增加了mock的配置
mock:{
  exclude:['mock/adv/summary','adv/summary','summary']
}

我想禁用 mock/adv/summary.ts 这个内部的mock，但是如上配置后任然还是在走mock的数据

**Comment by 1029800858** [comment]
请问解决了么，我项目Umi3也遇到了这个问题
